src/routes/community/components/newsPagination.jsx

```diff
@@ -0,0 +1,66 @@
+import * as N from '../news/NewsStyle';
+import { useState, useEffect } from 'react';
+import NewsItem from './newsItem';
```

Comment:
```suggestion
import NewsItem from '@community/components/newsItem';
```
해당 부분도 절대경로 추천드립니다 :)